packages/e2e-tests/src/specs/editor/media/insertWebMVideo.js

```diff
@@ -72,6 +72,7 @@ describe('Inserting WebM Video', () => {
     uploadedFiles.push(fileName);
 
     await page.waitForSelector('[data-testid="mediaElement-video"]');
+    await page.waitForTimeout(100);
```

Comment:
Note: it seems like without a small delay the correct video won't be available on time.
It's a bit hard to see what exactly is happening there since in interactive mode everything works as expected but I guess the new video will be otherwise added after the click happened already and that messes things up.
Not sure if there's a better way to fix these tests. Feel free to amend to the PR if you have ideas.

Reply:
One alternative could be clicking in the corner of the element to avoid the Insertion menu but not sure if that's the right approach either -- if we're waiting for the uploaded video.

Reply:
Ideally we'd use a more targeted `waitFor...` specifically for this new element.

The `await page.waitForSelector('[data-testid="mediaElement-video"]')`above is a bit pointless, as it is too generic and just waits for any video to exist in the library, which is always the case.

I'll try to find a more specific selector.